src/sidebar/helpers/query-parser.ts

```diff
@@ -134,6 +134,15 @@ export type Facet = {
    */
   operator: 'and' | 'or';
   terms: string[] | number[];
+
+  /**
+   * Whether this facet is relevant for replies.
+   *
+   * This is true for facets such as annotation bodies, which both annotations
+   * and replies have, but false for eg. page numbers, which only annotations
+   * have.
+   */
+  filterReplies: boolean;
```

Comment:
The values for this property are currently fixed for each facet. In future I can imagine we might want to customize it in some cases - eg. "filter for threads _started_ by user X" rather than "filter annotations written by user X".